docs/rootfs-and-kernel-setup.md

```diff
@@ -60,7 +60,7 @@ To build an EXT4 image:
    on how much data you'll want to fit inside:
    
    ```bash
-   dd if=/dev/zero of=/rootfs.ext4 bs=1M count=50
+   fallocate -l 50mb rootfs.ext4
```

Comment:
There's a small inaccuracy here: the doc says "50 MiB", but the command allocates 50 MB (i.e. 50 million bytes). Can you please update the command to use MiB (i.e. 50 * 1024 * 1024 bytes)?